diff --git a/library/src/tensile_host.cpp b/library/src/tensile_host.cpp
index dfb5644..e3b47b7 100644
--- a/library/src/tensile_host.cpp
+++ b/library/src/tensile_host.cpp
@@ -491,35 +491,7 @@ namespace
             }
             else
             {
-#ifndef ROCBLAS_STATIC_LIB
-                Dl_info info;
-
-                // Find the location of librocblas.so
-                // Fall back on hard-coded path if static library or not found
-                // [Use a void C API (rocblas_shutdown) *not* defined in this file to
-                // avoid compile-time resolution of the function pointer; cf.
-                // https://man7.org/linux/man-pages/man3/dladdr.3.html "BUGS"]
-                // rocblas_sscal stopped working even though is not defined in this unit
-
-                if(dladdr((void*)rocblas_shutdown, &info))
-                {
-                    path = info.dli_fname;
-                    path = std::string{dirname(&path[0])};
-                }
-                else
-#endif
-                {
-                    path = "/opt/rocm/rocblas/lib";
-                }
-
-                // Find the location of the libraries
-                if(TestPath(path + "/../../Tensile/library"))
-                    path += "/../../Tensile/library";
-                else
-                    path += "/library";
-
-                if(TestPath(path + "/" + processor))
-                    path += "/" + processor;
+                path="@GENTOO_PORTAGE_EPREFIX@/usr/lib64/rocblas/library";
             }
 
             // only load modules for the current architecture
